ux.symfony.com/src/Entity/LiveMemory/Game.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Entity\LiveMemory;
+
+use Symfony\Component\Uid\Uuid;
+use Symfony\Component\Uid\UuidV7;
+
+/**
+ * @demo LiveMemory
+ *
+ * @author Simon André <smn.andre@gmail.com>
+ */
+class Game
+{
+    private readonly string $id;
+
+    private readonly int $level;
+
+    private readonly string $theme;
+
+    private readonly int $timeLimit;
+
+    /**
+     * Map of game card as <cardKey> => <cardValue>.
+     *
+     * @var list<int, string>
+     */
+    private readonly array $cards;
+
+    /**
+     * @var array<int>
+     */
+    private array $matches = [];
+
+    /**
+     * @var list<int>
+     */
+    private array $flips = [];
+
+    private int $score = 0;
+
+    private int $timeBonus = 0;
+
+    private int $moveBonus = 0;
+
+    private ?\DateTimeImmutable $startedAt = null;
+
+    private ?\DateTimeImmutable $endedAt = null;
+
+    /**
+     * @param list<string> $cards List of card values
+     */
+    public function __construct(int $level, string $theme, array $cards, int $timeLimit)
+    {
+        $this->id = Uuid::v7()->toBase58();
+        $this->level = $level;
```

Comment:
Minor: should we show the player's current level somewhere? I had to dig a bit to see that you could level up after completing a round.

Reply:
Oh nice! Did not think about it, will find a way!

Reply:
I'll change later, but it's duly noted !